OMCompiler/SimulationRuntime/OMSICpp/runtime/include/Core/System/ExtendedSystem.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+/** @defgroup coreSystem Core.System
+ *  Core module for all algebraic and ode systems
+ *  @{
+ */
+/*****************************************************************************/
+/**
+
+Services, which can be used by systems.
+Implementation of standart functions (e.g. giveRHS(...), etc.).
+Provision of member variables used by all systems.
+
+Note:
+The order of variables in the extended state vector perserved (see: "Sorting
+variables by using the index" in "Design proposal for a general solver interface
+for Open Modelica", September, 10 th, 2008
+
+
+\date     October, 1st, 2008
```

Comment:
Is this an old file? Otherwise we should update the date.

Reply:
No this is a new class, I will update it.

Reply:
There are a few other places with the same stuff.